app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java

```diff
@@ -94,10 +98,11 @@ private void initComponent(View rootView) {
         mMapView = rootView.findViewById(R.id.mapView);
         sgwBtn = rootView.findViewById(R.id.SGWBtn);
         loyolaBtn = rootView.findViewById(R.id.loyolaBtn);
+        myLocation = rootView.findViewById(R.id.myLocation);
```

Comment:
can we call this myLocationBtn instead? Just for clarity when we are using this var elsewhere in the class.

Reply:
for sure.